src/conversions/indexmap.rs

```diff
@@ -1,3 +1,97 @@
+//!  Conversions to and from [indexmap](https://docs.rs/indexmap/)’s
+//! `IndexMap`.
+//!
+//! [`indexmap::IndexMap`] is a hash table that is closely compatible with the standard [`std::collections::HashMap`],
+//! with the difference that it preserves the insertion order when iterating over keys. It was inspired
+//! by Python's 3.6+ dict implementation.
+//!
+//! Dictionary order is guaranteed to be insertion order since Python 3.7, and the conversions with IndexMap will reflect
```

Comment:
Given that people don't need to worry about this, I probably wouldn't mention the order guarantees at all.